ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
+nodejs_app_git_repo: https://github.com/gpii/universal.git
+nodejs_app_git_branch: master
+nodejs_app_npm_packages:
+  - grunt-cli
```

Comment:
@gtirloni I'm OK with leaving it out if we're installing grunt-cli from the base nodejs role - it doesn't do any harm, but it's redundant.